src/test/__tests__/ReactTestUtils-test.js

```diff
@@ -19,286 +22,290 @@ var ReactTestUtils;
 
 describe('ReactTestUtils', () => {
   beforeEach(() => {
+    createRenderer = require('ReactShallowRenderer').createRenderer;
     PropTypes = require('prop-types');
     React = require('react');
     ReactDOM = require('react-dom');
     ReactDOMServer = require('react-dom/server');
     ReactTestUtils = require('ReactTestUtils');
   });
 
-  it('should have shallow rendering', () => {
-    class SomeComponent extends React.Component {
-      render() {
+  // Shallow renderer only implemented for Fiber in 16+
+  if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Oh. I didn't realize we *deleted* the Stack based one. Our usual strategy is to keep both until we’ve migrated everything that uses the old one (e.g. www). This lets us revert to using the old one if the new one has issues. Otherwise once this is merged, we have no way but to fix forward. Which might be fine, but it’s blocking any future syncs, and I don’t know if this will pass www tests or not.

Usually we keep both, and let the tests pick the right one based on the flag.